docs/faq.rst

```diff
@@ -65,27 +65,18 @@ I get import errors on libraries that depend on C modules
 
 This happens because our build system doesn't have the dependencies for building your project. This happens with things like libevent and mysql, and other python things that depend on C libraries. We can't support installing random C binaries on our system, so there is another way to fix these imports.
 
-You can mock out the imports for these modules in your ``conf.py`` with the following snippet::
 
-    import sys
-    from unittest.mock import MagicMock
+autodoc_mock_imports
```

Comment:
We could start this with something more explicit, like "You can use this feature included in autodoc to mock..." where the feature name would link to the sphinx's docs. Thanks for contributing btw :)